src/components/Proposals/ProposalDetail.js

```diff
@@ -17,11 +17,47 @@ import {
 } from '@aragon/ui'
 import { safeDiv } from '../../lib/math-utils'
 import { useGetVote } from '../../hooks/disputable-voting-logic'
+import {
+  DISPUTABLE_VOTE_STATUSES,
+  VOTE_STATUS_CANCELLED,
+  VOTE_STATUS_DISPUTED,
+  VOTE_STATUS_PAUSED,
+} from './disputable-vote-statuses'
 import DisputableActionStatus from './DisputableActionStatus'
 import InfoBoxes from './InfoBoxes'
 import SummaryBar from './SummaryBar'
+import FeedbackModule from './FeedbackModule'
 import Layout from '../Layout'
 import { networkEnvironment } from '../../current-environment'
+import { addressesEqual } from '../../lib/web3-utils'
+
+function getAttributes(status, theme) {
+  const attributes = {
+    [VOTE_STATUS_CANCELLED]: {
+      backgroundColor: theme.surfacePressed,
+      borderColor: theme.controlUnder,
+      disabledProgressBars: true,
+    },
+    [VOTE_STATUS_PAUSED]: {
+      backgroundColor: '#fffdfa',
+      borderColor: theme.warning,
+      disabledProgressBars: true,
+    },
+    [VOTE_STATUS_DISPUTED]: {
```

Comment:
I was just looking through the designs and wondering about the term 'disputed' now, it seems like some of the phrasing has changed to 'challenged'? do you think we should we update our naming to reflect this?